rules/windows/process_creation/proc_creation_win_susp_certutil_command.yml

```diff
@@ -22,16 +22,18 @@ detection:
             - ' -urlcache '
             - ' -verifyctl '
             - ' -encode '
+            - ' -exportPFX '
             - ' /decode '
             - ' /decodehex '
             - ' /urlcache '
             - ' /verifyctl '
             - ' /encode '
+            - ' /exportPFX '
     selection_certutil:
         Image|endswith: '\certutil.exe'
         CommandLine|contains: 'ping'
             # - 'URL'  # questionable string; could cause false positive
-    condition: 1 of selection_*
+    condition: all of selection_*
```

Comment:
With the new condition need "ping" in all CommandLine

Reply:
Yes, I've overlooked that. Nevertheless, if the old condition is used, the rule can match without "\certutil.exe" contained in the CommandLine. Is this intentional?

Reply:
It would be necessary to search in the history of commir from where comes this ping which does not have any sense.
Find this https://github.com/SigmaHQ/sigma/commit/04125cc4c093949352c9e7a3ae8cc459d74f5800
Seen in time 'certutil* -URL' become 'certutil' AND 'URL' ...
The rule detect certutil option not the certutil.exe

Reply:
I think we remove the second selection completely as the ping isn't that interesting in the first place or at least not with the same severity.
![image](https://user-images.githubusercontent.com/8741929/195168084-b22bb852-22e9-47f6-8e5f-7b4cb0649eeb.png)